### DISCUSSION TITLE
500 Server Error Each Sunday

### DISCUSSION THREAD

**Discussion by seh2000** [alexjustesen/speedtest-tracker#1068] [Q&A] [answered]
Hi,
I am having a 500 Server Error each Sunday.
Not sure how to fix, so each Sunday I do sudo docker-compose up -d and a reboot of the Raspberry Pi which makes the error go away. 
The app speedtest-tracker is running in a docker on a Raspberry Pi 4, where the app pihole is as well running.
I am using the below script to set the app up.
Any suggestions on how to solve the error? I have read about the HTTP 500 Internal Server Error, but fail to figure out if same error or two different errors. The strange part (for me) is for the whole time between Sundays the app is running without an issue.
Much appreciate - Steen

version: '3.3'
services:
    speedtest-tracker:
        container_name: speedtest-tracker
        ports:
            - '8981:80'
            - '8943:443'
        environment:
            - PUID=1000
            - PGID=1000
            - DB_CONNECTION=mysql
            - DB_HOST=db
            - DB_PORT=3306
            - DB_DATABASE=speedtest_tracker
            - DB_USERNAME=speedy
            - DB_PASSWORD=password
           #- TZ=Europe/Dublin
        volumes:
           #- '/etc/localtime:/etc/localtime:ro'
            - '/path/to/directory:/config'
            - '/path/to/directory/web:/etc/ssl/web'
        image: 'ghcr.io/alexjustesen/speedtest-tracker:v0.14.4'
        restart: unless-stopped
        depends_on:
            - db
    db:
        image: mariadb:10
        restart: always
        environment:
            - MARIADB_DATABASE=speedtest_tracker
            - MARIADB_USER=speedy
            - MARIADB_PASSWORD=password
            - MARIADB_RANDOM_ROOT_PASSWORD=true
        volumes:
            - speedtest-db:/var/lib/mysql
volumes:
  speedtest-db:

**Comment by alexjustesen** [comment, marked as answer]
This section of your config is incorrect, you'll need to map these either to a volume or location on the host machine. `500 Server Error` is letting you know "something" went wrong, you can pass `APP_DEBUG=true` as an environment variable to get more info.

```
- '/path/to/directory:/config'
- '/path/to/directory/web:/etc/ssl/web'
```

If you're not using a customer SSL cert you can remove the mount for `- '/path/to/directory/web:/etc/ssl/web'`.

**Reply by seh2000** [reply]
Hi Alex,
Many thanks...
Warm Regards - Steen
